src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -16,44 +16,45 @@ public class FindMyRecyclingController {
     @Autowired
     IFacilityService facilityService;
 
+    /* */
```

Comment:
Empty JavaDoc isn't adding any value.  It's clutter, which increases technical debt.